### ISSUE TITLE
[New Package]: Kingsoft.KDocs

### ISSUE THREAD

**Issue by SpecterShell** [vedantmgoyal2009/winget-pkgs-automation#119]
### List the PackageIdentifier of the package to be added to automation

Kingsoft.KDocs

### Details about the application

https://www.kdocs.cn/kdg/api/v1/configure?idList=kdesktopWinVersion

Parse `$result.data.kdesktopWinVersion` to JSON first, and then

`$inner_result.version` PackageVersion
`$inner_result.url` InstallerUrl

**Comment by SpecterShell** [comment]
Version API
https://www.kdocs.cn/kd/api/configure/list?idList=kdesktopVersion
```JSON
{
  "code": 0,
  "result": "OK",
  "data": {
    "kdesktopVersion": "{\"win\":\"v3.0.4\",\"mac\":\"v3.0.4\"}"
  }
}
```

Download API
https://www.kdocs.cn/kd/api/configure/list?idList=autoDownload
```JSON
{
  "code": 0,
  "result": "OK",
  "data": {
    "autoDownload": "{\"kdesktopWin\":{\"1001\":\"https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_1001_v3.0.4.exe\",\"1006\":\"https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_1006_v3.0.4.exe\",\"2009\":\"https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_2009_v3.0.4.exe\",\"1005\":\"https://qn.cache.wpscdn.cn/kdocs/apk/kdesktopwin/KDocs_1005_v3.0.4.exe\"},\"androidApp\":{\"cn00700\":\"https://qn.cache.wpscdn.cn/kdocs/apk/wpsyun_v2.13.0_2130_cn00700_release_2022.01.07_09.47.apk\",\"cn00706\":\"https://qn.cache.wpscdn.cn/kdocs/apk/wpsyun_v2.7.7_cn00706_release.apk\",\"cn00707\":\"https://qn.cache.wpscdn.cn/kdocs/apk/wpsyun_v2.6.7_cn00707_release.apk\"}}"
  }
}
```

InstallerUrl: `((irm 'https://www.kdocs.cn/kd/api/configure/list?idList=autoDownload').data.autoDownload | ConvertFrom-Json).kdesktopWin.1001`
PackageVersion: `((irm 'https://www.kdocs.cn/kd/api/configure/list?idList=kdesktopVersion').data.kdesktopVersion | ConvertFrom-Json).win` or from InstallerUrl

**Comment by vedantmgoyal2009** [comment]
Fixed in Automation v2.0.0.
